web/themes/custom/usagov/scripts/callCenterWaitTimeLookup.js

```diff
@@ -0,0 +1,85 @@
+jQuery(document).ready(async function () {
+  "use strict";
+
+  var waittime = (function() {
+    if (jQuery("html[lang|='en']").length ||
+        jQuery("html[lang|='es']").length) {
+      var domain = window.location.origin.split('.')[0].split('//')[1];
+      var domainMap = {
```

Comment:
I have really been trying to get us away from this! We should not be calling the CMS for this information. 

Instead, we should be making an https request of an s3 bucket (like the analytics dashboard's javascript does). 

There's also an implication here that we really should be running the call center cron jobs only on one system, since they will write data to a bucket everyone can share. It's fine for the code to be in all three environments and we should have a way to turn it on or off so we'll be able to test changes going forward. That could be as simple as a 1/0 environment variable on the app.